headers/functions/arm9.h

```diff
@@ -92,7 +92,7 @@ void AllocAndLoadFileInPack(enum pack_file_id pack_id, uint32_t file_index,
                             struct pack_alloc_and_load_result* output, uint32_t malloc_flags);
 void OpenPackFile(struct pack_file_opened* pack_file, const char* file_name);
 uint32_t GetFileLengthInPack(struct pack_file_opened* pack_file, uint32_t file_index);
-union faint_reason GetFaintReason(enum move_id, enum item_id);
+union damage_source GetFaintReason(enum move_id, enum item_id);
```

Comment:
Should the function be renamed as well?

Reply:
Fixed!
I also updated a few more things that I forgot to rename